ci/deploy.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+token=$CARGO_TOKEN
+tag=$TRAVIS_TAG
+options=""
+while [[ $# -gt 0 ]]; do
+	case $1 in
+		--token) token="$2"; shift;;
+		--tag) tag="$2"; shift;;
+		--dry-run) options+="--dry-run ";;
+		-v|--verbose) options+="--verbose ";;
+		-h|--help) cat << EOF
```

Comment:
Damn, this is some serious shell script magic.

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2F12NUbkX6p4xOO4%2Fgiphy.gif&token=magic" data-gitmeme-token="magic"><img src="https://media1.giphy.com/media/12NUbkX6p4xOO4/giphy.gif" title="Created by gitme.me with /magic"/></a>

Reply:
I copy-pasted it from the internetz

Reply:
Mind putting the link as a reference into the comments?

Reply:
> Mind putting the link as a reference into the comments?

https://stackoverflow.com/a/33826763